core/headers.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+package core;
+
+
+option go_package = "Rinnegan/proto-generated/core";
+
+// Request header message
+message RequestHeader {
+    string app_id = 1;       // Unique identifier for your app
+    AppVersion app_version = 2;  // Version of the app
+    string secure_token = 3; // Token to authenticate the request, Planning to take it as JWT Token
```

Comment:
Yes, if we want to have a JWT token, we can make a custom Message for that and then accept in that format instead of plain string

Reply:
A JWT token is a string only according to my understanding, if I am wrong let me know